transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/specification/extensions/pingpong/PingPongExtensionIT.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2014, Kaazing Corporation. All rights reserved.
```

Comment:
You'll need to update this copyright header once you put in the latest changes from upstream develop (just copy the header of any other java class).